### PR TITLE
DM-47199: `idfprod` Sasquatch

### PR DESCRIPTION
**Pull request by fajpunk**

### CODE REVIEW

File: applications/sasquatch/values-idfprod.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  #   source:
+  #     bootstrapServer: sasquatch-dev-kafka-bootstrap.lsst.cloud:9094
+  #     topicsPattern: "registry-schemas, lsst.sal.*, lsst.dm.*"
+  #   resources:
+  #     requests:
+  #       cpu: 2
+  #       memory: 4Gi
+  #     limits:
+  #       cpu: 4
+  #       memory: 8Gi
```
Comment (fajpunk) [inline applications/sasquatch/values-idfprod.yaml:31]: Not needed since we're not replicating

Comment (fajpunk) [inline applications/sasquatch/values-idfprod.yaml:31]: Not needed since we're not replicating

File: applications/sasquatch/values-idfprod.yaml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  kraft:
+    enabled: true
+  kafkaController:
+    enabled: true
+    resources:
+      requests:
+        # memory: 16Gi
+        memory: 8Gi
+        # cpu: "2"
+        cpu: "1"
```
Comment (fajpunk) [inline applications/sasquatch/values-idfprod.yaml:54]: For this and other resource limits and requests, I don't think this needs to be as big as the other installations given that it's only going to handle app metrics.

File: applications/sasquatch/values-idfprod.yaml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    GENERIC_NAME: "OIDC"
+    GENERIC_AUTH_URL: https://data.lsst.cloud/auth/openid/login
+    GENERIC_TOKEN_URL: https://data.lsst.cloud/auth/openid/token
+    USE_ID_TOKEN: 1
+    JWKS_URL: https://data.lsst.cloud/.well-known/jwks.json
+    GENERIC_API_URL: https://data.lsst.cloud/auth/openid/userinfo
+    GENERIC_SCOPES: openid
+    GENERIC_API_KEY: sub
+    PUBLIC_URL: https://data.lsst.cloud/
+    # STATUS_FEED_URL: https://raw.githubusercontent.com/lsst-sqre/rsp_broadcast/main/jsonfeeds/idfint.json
```
Comment (fajpunk) [inline applications/sasquatch/values-idfprod.yaml:171]: Should I make an `idfprod.json` file in `rsp_broadcast`?

**Comment by fajpunk** [comment]
@afausti This is successfully synced, thanks for your help! I've left comments in for what I've changed from the `idfint` config. If this looks good to you, I'll remove the commented-out config before merging.

File: applications/sasquatch/values-idfprod.yaml (hunk 2 above)
Comment (afausti) [reply applications/sasquatch/values-idfprod.yaml:54]: Sounds good, we can start with that.

File: applications/sasquatch/values-idfprod.yaml (hunk 3 above)
Comment (afausti) [reply applications/sasquatch/values-idfprod.yaml:171]: Yes, that's the mechanism we use to populate the Chronograf news feed section, in the first page after you log into Chronograf. Can be used for notifications about this deployment.

**Review by afausti — APPROVED** [review]
Looks good to me!

**Pull request merged**